contrib/debian/copyright

```diff
@@ -5,9 +5,9 @@ Upstream-Contact: Satoshi Nakamoto <satoshin@gmx.com>
 Source: https://github.com/bitcoin/bitcoin
 
 Files: *
-Copyright: 2009-2012, Bitcoin Core Developers
+Copyright: 2009-2012, Bitcoin Classic Developers
 License: Expat
-Comment: The Bitcoin Core Developers encompasses the current developers listed on bitcoin.org,
+Comment: The Bitcoin Classic Developers encompasses the current developers listed on bitcoin.org,
```

Comment:
I dont believe we have a full site yet, once we do these will be changed. i requested that all url's be changed in a seperate PR